src/containers/paper-canvas.jsx

```diff
@@ -114,13 +124,31 @@ class PaperCanvas extends React.Component {
             this.props.canvasRef(canvas);
         }
     }
+    handleWheel (event) {
```

Comment:
Why is this called 'handleWheel'? Does it not work on touch pads and touch screens?

Reply:
Yup, the event is called `onWheel` so convention dictates the handler should be called `handleWheel`. The event is emitted by both mouse wheels and trackpads, but not on touch devices. We will need to handle those differently.